pages/style-guide/html.md

```diff
@@ -4,64 +4,162 @@
 
 This page outlines the style guide for HTML pages in all jQuery projects. These rules apply to web sites, demo pages, inline examples, test pages, etc. Exceptions are allowed for pages that must violate the rules by their very nature, e.g., a page that tests XHTML.
 
-## Doctype
-Always use the minimal, versionless doctype.
+## Linting
+
+The markup should be valid HTML5 and it must pass the automated testing we use via grunt-html.
+
+## Spacing
+
+In general, the jQuery style guide encourages liberal spacing for improved human readability. The minification process creates a file that is optimized for browsers to read and process.
+
+- Identation only with tabs where one tab equals 4 spaces. Don't indent the first nested elements of `html`, `body`, `script`, or `style`. Indent everything else.
```

Comment:
> Identation only with tabs where one tab equals 4 spaces.

I don't like the word `equals` in here. To me it looks like we use 4 spaces. Needs clarification.